packages/sucrase/src/index.js

```diff
@@ -15,12 +15,21 @@ module.exports = function sucrase(opts = {}) {
       if (importer && /^[./]/.test(importee)) {
         const resolved = path.resolve(importer ? path.dirname(importer) : process.cwd(), importee);
         // resolve in the same order that TypeScript resolves modules
-        const resolvedFilename = [
+        const resolvedFilenames = [
           `${resolved}.ts`,
           `${resolved}.tsx`,
           `${resolved}/index.ts`,
           `${resolved}/index.tsx`
-        ].find((filename) => fs.existsSync(filename));
+        ];
+        if (importee.endsWith('.js')) {
```

Comment:
```suggestion
        if (resolved.endsWith('.js')) {
```

should we use resolved here too?

Reply:
Sure why not, it shouldn't be any different. Feel free to commit this suggestion to my branch.